README.md

```diff
@@ -125,6 +125,10 @@ system-period temporal tables behaviour:
 
     versioning(<system_period_column_name>, <history_table_name>, <adjust>)
 
+It is possible to use a spécific shema, even the extension was created in the public schéma :
```

Comment:
You seem to have a couple `é` characters where you just want `e`

Also `s/shema/schema/`